packages/cli/src/Server.ts

```diff
@@ -382,6 +386,36 @@ export class Server extends AbstractServer {
 		}
 	}
 
+	async configureProxyEndpoints(): Promise<void> {
+		const licenseService = Container.get(License);
+		const licenseCert = await licenseService.loadCertStr();
+		const authService = Container.get(AuthService);
+		const consumerId = licenseService.getConsumerId();
+		const apiRestPath = Container.get(GlobalConfig).endpoints.rest;
+		const aiAssistantEnabled = licenseService.isAiAssistantEnabled();
+
+		const aiServiceApiBase = config.get('aiAssistant.baseUrl');
```

Comment:
Any reason why are we using `config.get` only here, and not `Container.get(GlobalConfig)`?

Reply:
Because the rest path in the globalConfig object and the `aiAssistant.baseUrl` is not. We are currently in the middle of a migration to the GlobalConfig object.